File: src/practice/luogu/p5720/Main.cpp
// https://www.luogu.com.cn/problem/P5720
#include <cstdio>

int main() {
  int n;
  scanf("%i", &n);
  int i = 0;
  while (n > 1) {
    n /= 2;
    i++;
  }
  printf("%i\n", i + 1);
  return 0;
}

File: src/practice/luogu/p5720/Main.php
<?php
// https://www.luogu.com.cn/problem/P5720
fscanf(STDIN, '%d', $n);
$i = 0;
while ($n > 1) {
  $n = (int)($n / 2);
  $i++;
}
echo $i + 1;
